.github/workflows/publish.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       uses: pypa/cibuildwheel@v2.21.3
       env:
         CBOR2_BUILD_C_EXTENSION: "1"
-        CIBW_SKIP: pp* cp36* cp37*
+        CIBW_SKIP: pp*
```

Comment:
cibuildwheel looks at `requires-python` in `pyproject.toml` so we don't need to explicitly list the old ones here.

https://cibuildwheel.pypa.io/en/stable/options/#requires-python

Reply:
Perhaps that has changed in a while because I added these specifically to block the creation of wheels for these versions after testing first. Or I may misremember and maybe I just added these based on a potential misunderstanding of their docs.

Reply:
I have some vague recollection of that too.

This feature was added in https://github.com/pypa/cibuildwheel/pull/536 which was released in https://github.com/pypa/cibuildwheel/releases/tag/2.5.0 in 2022, so not that long ago.